### PR TITLE
Update tabs and spacing

### PR DESCRIPTION
**Pull request by pdadhaniya**
@shreya317

### CODE REVIEW

File: app/views/guests/show.html.erb [hunk 1]
```diff
@@ -9,35 +9,34 @@
...
+          <input type="submit" value="Submit">
         </div>
-        <hr>
-        <div class= "location">
-            <b><h4 class= "location">Patel Residence</h4></b>
-            <h6 class= "location">9443 Lotus Ave.</h6>
-            <h6 class= "location">Skokie, IL 60077</h6>
-            <h6 class= "location">10:30 AM</h6>
+        <div class="large-6 medium-6 columns small-6">
+         <!-- <input type="submit" value="Submit"> -->
```
Comment (pdadhaniya) [inline app/views/guests/show.html.erb]: Can we remove this whole div?

**Comment by shreya317** [comment]
:100:

File: app/views/guests/show.html.erb (hunk 1 above)
Comment (shreya317) [inline app/views/guests/show.html.erb]: @pdadhaniya just remove the `<input>` please

Comment (shreya317) [inline app/views/guests/show.html.erb]: @pdadhaniya actually just remove the ```<h2>``` please

Comment (shreya317) [inline app/views/guests/show.html.erb]: @pdadhaniya Do you want to take care of the minor change above and merge it in after?

Comment (pdadhaniya) [inline app/views/guests/show.html.erb]: :+1:

**Pull request closed without merging**